<?xml version="1.0" encoding="UTF-8"?>
<launch>
   <!-- Topic names -->
   <param name="cmd_vel" value="cmd_vel" />
   <param name="scan" value="scan" />
   <param name="goal" value="goal" />
   <param name="odom" value="odom" />
   <!-- Velocity limits -->
   <param name="go_vw" value="0.2" />
   <param name="go_vx" value="1.0" />
   <param name="turn_vw" value="0.7" />
   <!-- Run bug planner -->
   <node pkg="bug_planning" type="follow_wall.py" name="wall_follower" />
   <node pkg="bug_planning" type="go_to_point.py" name="go_to_point" />
   <node pkg="bug_planning" type="bug2.py" name="bug2" output="screen" />
</launch>
